lib/ansible/modules/files/acl.py

```diff
@@ -71,8 +71,24 @@
     description:
       - Recursively sets the specified ACL (added in Ansible 2.0). Incompatible with C(state=query).
     type: bool
-    default: 'no'
-    version_added: "2.0"
+    default: no
+    version_added: '2.0'
+
+  use_nfsv4_acls:
+    description:
+      - Use NFSv4 ACLs instead of POSIX ACLs.
+    type: bool
+    default: no
+    version_added: '2.2'
+
+  recalculate_mask:
```

Comment:
This is a 2.7 feature that doesn't exist in 2.6 and shouldn't be included in the backport.